internal/log/log.go

```diff
@@ -117,36 +117,55 @@ func PrepareLogging(fs afero.Fs, verbose *bool, loggerSpy io.Writer) {
 		loglevel = loggers.LevelDebug
 	}
 
-	logFile, err := prepareLogFile(fs)
+	logFile, errFile, err := prepareLogFiles(fs)
 	logFormat := loggers.ParseLogFormat(os.Getenv(loggers.EnvVarLogFormat))
 	logTime := loggers.ParseLogTimeMode(os.Getenv(loggers.EnvVarLogTime))
 
 	setDefaultLogger(loggers.LogOptions{
-		File:               logFile,
-		FileLoggingJSON:    logFormat == loggers.LogFormatJSON,
-		ConsoleLoggingJSON: logFormat == loggers.LogFormatJSON,
-		LogLevel:           loglevel,
-		LogSpy:             loggerSpy,
-		LogTimeMode:        logTime,
+		File:        logFile,
+		ErrorFile:   errFile,
+		JSONLogging: logFormat == loggers.LogFormatJSON,
+		LogLevel:    loglevel,
+		LogSpy:      loggerSpy,
+		LogTimeMode: logTime,
 	})
 
 	if err != nil {
 		Warn(err.Error())
 	}
 }
 
-func prepareLogFile(fs afero.Fs) (afero.File, error) {
-	timestamp := timeutils.TimeAnchor().Format(TrafficLogFilePrefixFormat)
+// LogFilePath returns the path of a logfile for the current execution time - depending on when this function is called such a file may not yet exist
+func LogFilePath() string {
+	timestamp := timeutils.TimeAnchor().Format(LogFileTimestampPrefixFormat)
+	return filepath.Join(LogDirectory, timestamp+".log")
+}
+
+// ErrorFilePath returns the path of an error logfile for the current execution time - depending on when this function is called such a file may not yet exist
+func ErrorFilePath() string {
+	timestamp := timeutils.TimeAnchor().Format(LogFileTimestampPrefixFormat)
+	return filepath.Join(LogDirectory, timestamp+".errors")
```

Comment:
I'd go with `{timestamp}-errors.log`. `.errors` might be a more confusing file suffix

Reply:
Fair point, renamed it as suggested